jobserver/models/core.py

```diff
@@ -837,6 +836,9 @@ class User(AbstractBaseUser):
     pat_token = models.TextField(null=True, unique=True)
     pat_expires_at = models.DateTimeField(null=True)
 
+    # this token is generated for users when they want to log in
+    login_token = models.TextField(null=True, unique=True)
```

Comment:
I don't think we need to store the token in db any more - we can just add it to the session.

That way there's no other way except via the session.